go/engine/pgp_keygen_test.go

```diff
@@ -47,3 +47,42 @@ func TestPGPKeyGen(t *testing.T) {
 		t.Errorf("result keys: %d, expected 1", len(xe.Results()))
 	}
 }
+
+func TestPGPKeyGenNoPush(t *testing.T) {
+	tc := SetupEngineTest(t, "pgpkeygen")
```

Comment:
Why not make the test above have a loop that goes through shouldPush [true, false]?

Not saying you should change this just curious